packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.1]
+## Changed
```

Comment:
This fixes the lint error:
```suggestion
### Changed
```

Reply:
Ah, good catch. I fixed this in eb56aaa97c487794c0a3c2bd51affdd99688380d.